.github/workflows/basic.yml

```diff
@@ -12,13 +12,18 @@ jobs:
         build_type: [Debug, Release]
         compiler: [{c: gcc, cxx: g++}]
         pool_tracking: ['-DUMF_ENABLE_POOL_TRACKING=ON', '-DUMF_ENABLE_POOL_TRACKING=OFF']
+        shared_library: ['-DUMF_BUILD_SHARED_LIBRARY=OFF']
```

Comment:
Could you simplify those options like in https://github.com/oneapi-src/unified-memory-framework/pull/87/files ?

Reply:
done